domainhunter.py

```diff
@@ -18,6 +18,7 @@
 import sys
 import urlparse
 import getpass
+import uuid
```

Comment:
requirements.txt must be updated to include new packages `urlparse` `getpass` `uuid`